src/index.html

```diff
@@ -6,9 +6,24 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <ul class="nav_link" >Video</ul>
+        <ul class="nav_link" >Photo</ul>
+        <ul class="nav_link" >Tablets</ul>
+        <ul class="nav_link" >Gadgets</ul>
+        <ul class="nav_link" data-qa="hover">Laptops & Computers</ul>
+        <ul class="nav_link" >Smartphones</ul>
+        <ul class="nav_link" >Samsung</ul>
+        <ul class="nav_link nav_link1"><a href="index.html" class="is-active">Apple</a></ul>
+      </nav>
+      <a href="index.html" class="home_page"><img src="images/logo.png" alt="home_page"></a>
```

Comment:
The logo image seems to be outside of the 'nav' element. Usually, the logo is part of the navigation bar.

Reply:
readme.md 
`Logo should also be a link to the main page of the website (#home). But it should not be part of nav`